decode.go

```diff
@@ -111,11 +111,6 @@ func (d *Decoder) Seek(offset int64, whence int) (int64, error) {
 	return npos, nil
 }
 
-// Close is io.Closer's Close.
-func (d *Decoder) Close() error {
-	return d.source.Close()
```

Comment:
Could you keep this function with empty implementation not to break the compatibility? I'd want a comment like `Close is io.Closer's Close. Close does nothing and always returns nil.`

Reply:
Ok, I will add it, but once you migrate to modules, I would strongly recommend to delete it for next release.

Reply:
Good point. I'll do when I release 0.2.0.